src/api/spec/controllers/webui/staging/workflows_controller_spec.rb

```diff
@@ -119,9 +119,10 @@
   end
 
   describe 'DELETE #destroy' do
+    let!(:staging_workflow) { create(:staging_workflow, project: project) }
```

Comment:
I think if you do it like this you need to use `staging_workflow_with_staging_projects`. :thinking:

Reply:
You're right, fixed